src/helpers/dimensions.ts

```diff
@@ -155,6 +155,56 @@ function getBarDimensions(mdObject: VisualizationObject.IVisualizationObjectCont
     ];
 }
 
+function getAreaDimensions(mdObject: VisualizationObject.IVisualizationObjectContent): AFM.IDimension[] {
+    const view: VisualizationObject.IBucket = mdObject.buckets
+            .find(bucket => bucket.localIdentifier === VIEW);
+
+    const stack: VisualizationObject.IBucket = mdObject.buckets
+        .find(bucket => bucket.localIdentifier === STACK);
+
+    const hasNoStacks = !stack || !stack.items || stack.items.length === 0;
+    const hasManyViewItems = view && view.items && view.items.length > 1;
+
+    if (hasManyViewItems) {
+        // only take first two view items
+        const [firstViewItemIdentifier, secondViewItemIdentifier] = view.items
+            .slice(0, 2)
```

Comment:
You can use `VIEW_BY_ATTRIBUTES_LIMIT` in `/gooddata-react-components/src/components/visualizations/chart/constants.ts`

Reply:
how about renaming vars 
from 
`const [firstViewItemIdentifier, secondViewItemIdentifier] = ...`
to
`const [viewByItemIdentifier, stackByItemIdentifier] = ...;`